documentation/modules/exploit/linux/http/grandstream_ucm62xx_sendemail_rce.md

```diff
@@ -0,0 +1,220 @@
+## Vulnerable Application
+
+### Description
+
+This module exploits unauthenticated SQL and command injection vulnerabilities affecting
+the Grandstream UCM62xx IP PBX series of devices. The vulnerabilities allow an
+unauthenticated and remote attacker to execute commands as root. This module affects
+UCM62xx versions before firmware version 1.0.19.20.
+
+### Installation
+
+The UCM62xx PBX is a physical device and is not known to be succesfully emulated.
+However, if you have a device, affected firmware can be downloaded here:
+
+* https://firmware.grandstream.com/Release_UCM62xx_1.0.18.13.zip
+
+## Verification Steps
+
+* Acquire an affected device and configure with affected firmware
+* Do: `use exploit/linux/http/grandstream_ucm62xx_sendemail_rce`
+* Do: `set RHOST <ip>`
+* Do: `check`
+* Verify the remote target is flagged as vulnerable
+* Do: `set LHOST <ip>`
+* Do: `set LPORT <port>`
+* Do: `exploit`
+* You should get a Meterpreter session.
+
+## Targets
+
+### 0
+
+By default, this targets the PBX with the `reverse_netcat_gaping` payload and returns a reverse shell.
+
+### 1
+
+By default, this target obtains a meterpreter session using `wget`.
```

Comment:
Why the `by default` part here? What is the other case that may occur? If there is one, it should be explained here. If not, then we shouldn't be using the `by default` wording here and should just state what it does.